bees/events.go

```diff
@@ -51,7 +52,8 @@ func handleEvents() {
 		log.Println()
 		log.Println("Event received:", event.Bee, "/", event.Name, "-", GetEventDescriptor(&event).Description)
 		for _, v := range event.Options {
-			log.Println("\tOptions:", v)
+			vv := truncateString(fmt.Sprintln(v), 1000)
```

Comment:
I like the change, but as a general advice I would recommend moving unrelated changes into a separate commit & PR, to keep them as granular & atomic as possible. This makes it a lot easier to maintain code in the long run.

Reply:
Thanks, I'll keep that in mind